tests/Unit/Infrastructure/Twig/AppExtensionTest.php

```diff
@@ -44,71 +44,72 @@ public function testFormatDateTimeWithTime(): void
         );
     }
 
-    private function provideIsFuture(): array
+    private function provideIsPast(): array
     {
         return [
             // Day after
             [
                 'date' => '2023-01-07',
-                'time' => null,
-                'now' => '2023-01-06 10:30:00',
-                'result' => true,
+                'now' => '2023-01-06',
+                'result' => false,
             ],
+            // Day before
             [
-                'date' => '2023-01-07',
-                'time' => '09:00:00',
-                'now' => '2023-01-06 10:30:00',
+                'date' => '2023-01-05',
+                'now' => '2023-01-06',
                 'result' => true,
             ],
+            // Same day
+            [
+                'date' => '2023-01-06',
+                'now' => '2023-01-06',
+                'result' => false,
+            ],
```

Comment:
On peut ajouter un cas "que se passe-t-il si une heure est présente"

```suggestion
            // Same day (time does not matter)
            [
                'date' => '2023-01-06T11:30:00',
                'now' => '2023-01-06T08:00:00',
                'result' => false,
            ],
```